### ISSUE TITLE
Python script sends \r chars to much

### ISSUE THREAD

**Issue by CLWcynthia** [extrabacon/python-shell#93]
how to delete \r chars in output result?

my test.js like this

```
 	var PythonShell = require('python-shell');
 	var meta_str = JSON.stringify(meta);
 	meta_str = meta_str.replace(/\"/g, '\'');		// 把json字符串中的双引号改成单引号
 	var opt = {
 		pythonOptions: ['-u'],
 		scriptPath: __dirname +'/',
    	args:[filepath,meta_str]
	};
 	PythonShell.run('init.py',opt, function (err,results) {
	 	console.log("45");
		console.log(results)
		console.log(typeof(results))
    if (err) throw err;
    	console.log("49");
    	console.log(err);
    res.send("ok");
});
```

the results is
![image](https://user-images.githubusercontent.com/30431718/29554257-03937118-8751-11e7-8262-2b62bbacc585.png)

**Comment by atikuxdntz** [comment]
You just need to remove the last character (just the last character, not the last 2 because \r is considered just one) from the string, like this:
`result=result[:-1]`

**Comment by jdforsythe** [comment]
Change to unix line endings. This is because you have `\r\n` (Windows line endings) instead of just `\n`

**Comment by Almenon** [comment]
fixed with #109 https://github.com/extrabacon/python-shell/commit/ed7036cc53dab4ce191fed85633eed731a8367b1
